src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -253,7 +253,7 @@ def create_feedback_issue(event, project_id: int, source: FeedbackCreationSource
         payload_type=PayloadType.OCCURRENCE, occurrence=occurrence, event_data=event_fixed
     )
     if is_message_spam:
-        auto_ignore_spam_feedbacks(project_id, issue_fingerprint)
+        auto_ignore_spam_feedbacks(project, issue_fingerprint)
```

Comment:
is it safe to interchange `project` with `project_id`?

Reply:
i changed function signature below!

Reply:
ahh i missed that